bin/tessel-2.js

```diff
@@ -116,7 +121,7 @@ parser.command('crash-reporter')
     flag: true,
     help: 'Test the Crash Reporter.'
   })
-  .help('Crash Reporter enable/disable');
+  .help('See Crash Reporter status');
```

Comment:
It seems like this help text needs to be a little more general because you can also set the status in addition to seeing it. Maybe it should be "View or set Crash Reporter status" or something to that effect.